benchmarks/rtb_cache_benchmarks.cpp

```diff
@@ -14,7 +14,7 @@ namespace po = boost::program_options;
 #include "../examples/bidder/ad.hpp"
 #include "../examples/bidder/geo.hpp"
 #include "../examples/bidder/geo_ad.hpp"
-#include "../examples/bidder/campaign_data.hpp"
+//#include "../examples/bidder/campaign_data.hpp"
```

Comment:
I completely agree , will add campaign_budget instead of deprecated campaign_data as campaign_budget was never tested in our benchmarks. Watch for the changes , more changes are commit. Thanks for your help.